wp-content/themes/headstrong/css/styles.030.css

```diff
@@ -26,6 +26,10 @@
   font-weight: bold;
 }
 
+.header__jump {
```

Comment:
If we decide to create a new modifier for the header class, can we change this to .header--jump, because it's a modifier, not an element?